rfcs/new-coc-and-moderation.md

```diff
@@ -0,0 +1,92 @@
+against our guidelines", and it is reasonable to expect the other person to
+simply edit their message, offer a brief apology, and allow everyone to move on.
+This interaction should be common, comfortable, and rarely lead to escalation.
+
+6. **Trust The Ouch** - It's completely okay if you don't understand someone's
+reaction to something you said. If they say it hurts, start from the assumption
+that they are being honest and mean well, and offer your cooperation in
+adjusting what needs to be adjusted. We can never fully understand everyone
+around us. We can never learn every little thing that might upset or harm
+others. What we **can** do is listen when they tell you so, and cooperate with
+them from there. You're not a bad person for a habit slipping or unintentionally
+crossing a line you didn't know about.
+
+7. **Protection, Not Safety** - A project at any significant scale cannot be a
+safe space. Something, at some point, will happen, and it's important to make it
+clear to current and potential collaborators that this cannot be prevented.
+Instead, the project should be committed to going through reasonable,
+responsible processes whenever something **does** happen to alleviate the
+damage, and to make it clear to others that violations will be taken seriously.
+That is really the best a project would be able to offer.
+
+### Moderation Policy Changes
+
+Concretely, moving to a policy that the author believes fulfills these values
+involves a few steps:
+
+1. Replace the current code of conduct with one based on the [WeAllJS Code of
+Conduct](https://npm.im/weallbehave), which has a built-in enforcement process
+that promotes individual interactions, trusting the "ouch", and prioritizes the
+hurt over those who caused the hurt when the enforcement process is applied.
+
+2. Remove the existing Moderation Policy, since moderation details are now built
+into the CoC.
+
+3. Update relevant contact/reporting information for the moderation team to the CoC.
+
+4. Add a detail about using `<describe>` and `<summary>` when editing posts by
+collaborators, instead of editing in place, as well as providing a rational for
+the edit, and signing it.
+
+5. Add an event-tracking process that includes methods for tracking details
+about enforcement measures in private (managed by the Moderation Team), as well
+as publishing a scrubbed version of that log that is available to the public, as
+well as establishing a process for non-Moderators to request information kept in
+the private log. (Note: This might be worth having as its own RFC, but I think
+it's important)
+
+6. Specify the Moderation Team as the ultimate arbiter of moderation actions,
+but that changes to the moderation policy should be proposed through the
+standard RFC process, and must be ratified by both the Core and Moderation
+teams, at least, as well as by any other Team who believes the changes will
+affect them.
```

Comment:
Is there a future possibility here for a team working to block a policy change they don't like, particularly if it's made in response to a team's behavior? Or would the moderation team's authority kick in prior to this?

Reply:
Right now, since this is about inter-Team disagreements, I believe this may involve spawning off an [EFA team](https://github.com/ayojs/ayo/blob/latest/GOVERNANCE.md#elected-final-arbiters). I'm not sure whether to specify that as the process or leave it to the default?

Reply:
@Harrison-M 

We discussed this a lot on the Discord channel and we ended up leaving the language about how to spawn the EFA team intentionally vague to avoid unnecessary ceremony and overhead on getting up and running. We had two ideas for when consensus couldn't be met on an issue or RFC:

1. We elect a team of EFAs to serve for a predetermined term-length
2. We select a set of EFAs on an as-needed basis

Obviously neither of those solutions is fully fleshed out ("we" who?, "a set"? "team"? "term-length"?), but I think we're leaning toward the latter and we'll discover how that will come about in time, either through needing to make it happen, or having the bandwidth to give it a proper RFC.

Reply:
Anyone opposed to me leaving the language about this in _this_ RFC relatively vague and letting a different RFC go into specifics?